nix/cardano-services/deployments.nix

```diff
@@ -0,0 +1,57 @@
+let
+  inherit (inputs.std) dmerge;
+  inherit (inputs.std.inputs) haumea;
+  inherit (inputs.std.lib.ops) readYAML;
+
+  loadYaml = _: _: readYAML;
+
+  baseline = with haumea.lib;
+    load {
+      src = ./deployments;
+      loader = [(matchers.regex ''^.+\.(yaml|yml)'' loadYaml)];
+    };
+in {
+  dev-preview = dmerge baseline {
+    meta.description = "Development Environment on the Cardano Preview Chain (Frankfurt)";
+    backend-deployment = {
+      spec.template.spec.containers = dmerge.updateOn "name" [
+        {
+          name = "backend";
+          image = cell.oci-images.cardano-services.image.name;
+        }
+      ];
+    };
+    coingecko-proxy-deployment = {
+      spec.template.spec.containers = dmerge.updateOn "name" [
+        {
+          name = "coingecko-proxy";
+          image = cell.oci-images.cardano-services.image.name;
+        }
+      ];
+    };
+    handle-projector-deployment = {
+      spec.template.spec.containers = dmerge.updateOn "name" [
+        {
+          name = "handle-projector";
+          image = cell.oci-images.cardano-services.image.name;
+        }
+      ];
+    };
+    pgboss-deployment = {
+      spec.template.spec.containers = dmerge.updateOn "name" [
+        {
+          name = "pg-boss-worker";
+          image = cell.oci-images.cardano-services.image.name;
+        }
+      ];
+    };
+    stake-pool-projector-deployment = {
```

Comment:
Missing stake-pool provider and blockfrost services

Reply:
Partially addressed in [force-push](https://github.com/input-output-hk/cardano-js-sdk/compare/10167f20f9c7d8e8870f84755ab2e0bd1a902e2f..47ce6e3ec1f74f224a253a6c05706323d0dd3ba9)

Reply:
What was the matter with blockfrost service? Did I still miss something?

Reply:
yes, you missed a deployment